src/globus_cli/commands/timer/create.py

```diff
@@ -0,0 +1,288 @@
+import datetime
+from typing import List, Optional, Tuple, cast
+
+import click
+from globus_sdk import TimerJob, TransferData
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, group, mutex_option_group
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+
+from ._common import (
+    DATETIME_FORMATS,
+    JOB_FORMAT_FIELDS,
+    START_HELP,
+    parse_timedelta,
+    read_csv,
+    require_activated_endpoints,
+)
+
+INTERVAL_HELP = """
+    Interval at which the job should run. Use 'w', 'd', 'h', 'm', and 's' as suffixes
+    to specify weeks, days, hours, minutes, and seconds. Examples: '1h 30m', '500s',
+    '24h', '1d 12h', '2w', etc. Must be in order: hours -> minutes -> seconds. You
+    should either use quotes ('1d 2h') or write without spaces (1d2h).
+"""
+
+
+@group("create", short_help="Submit a Timer job", hidden=True)
+def create_command():
+    pass
+
+
+@command("transfer", short_help="Create a recurring transfer job in Timer")
+@click.option(
+    "--name",
+    required=True,
+    type=str,
+    help="Name to identify this job in the timer service (not necessarily unique)",
+)
+@click.option(
+    "--start",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=START_HELP,
+)
+@click.option(
+    "--interval",
+    required=False,
+    type=str,
+    help=INTERVAL_HELP,
+)
+@click.option(
+    "--source-endpoint",
+    required=True,
+    type=str,
+    help="ID for the source transfer endpoint",
+)
+@click.option(
+    "--dest-endpoint",
+    required=True,
+    type=str,
+    help="ID for the destination transfer endpoint",
+)
+@click.option(
+    "--label",
+    required=False,
+    type=str,
+    help=(
+        "An optional label for the transfer operation, up to 128 characters long. Must"
+        " contain only letters/numbers/spaces, and the following characters: - _ ,"
+    ),
+)
+@click.option(
+    "--stop-after-date",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=("Stop running the transfer after this date"),
+)
+@click.option(
+    "--stop-after-runs",
+    required=False,
+    type=int,
+    help=("Stop running the transfer after this number of runs have happened"),
+)
+@click.option(
+    "--sync-level",
+    required=False,
+    type=int,
```

Comment:
Rather than using an `int` and requiring that users learn the Transfer enum, can we share code with `globus transfer` for this?
https://github.com/globus/globus-cli/blob/b16ef4c18fcfbfa1c76136209ba76b3223523480/src/globus_cli/commands/transfer.py#L122